NEWS

```diff
@@ -76,6 +76,10 @@ Release a.b
    header `@PG` records. Thanks to John Marshall.  (#1394; reported by
    Kemin Zhou in #1393)
 
+ * Ampliconclip and ampliconstats now guard against the BED file
```

Comment:
I am no ampli-expert, but what do you mean by “reference” here? GRCh37 vs 38? Or reference chromosomes, i.e., more than one different value in the CHROM column?

Reply:
Reference as it RNAME.    Terminology is such a mess!  SAM calls them references so that's what I used.  VCF calls them contigs.  I'm assuming from your comment that BED calls them chromosomes? :/

Reply:
Messy indeed :smile: Yep, the documents that have historically served as specs say stuff like “`chrom` — The name of the chromosome or scaffold”. (I reckon SAM calls them “reference sequences”, sometimes shortened to “references”.) Thanks for the clarification applied.